### PR TITLE
tickets/PREOPS-1030

### PR DESCRIPTION
**Pull request by MelissaGraham**
Updated the "DP0.2 DPDD" page. Removed old schema. Assign to Jeff b/c PREOPS-1231 is related.

### CODE REVIEW

File: data-products-dp0-2/index.rst [hunk 1]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
+     - Butler Name
+     - Description
+   * - Object
+     - objectTable
+     - Astrometric and photometric measurements for objects detected in coadded images (990 columns).
+   * - Source
+     - sourceTable
+     - Astrometric and photometric measurements for sources detected in the individual PVIs (143 columns).
+   * - ForcedSource
+     - forcedSourceTable
```
Comment (jeffcarlin) [inline data-products-dp0-2/index.rst:128]: Also, there are `forcedSourceTable` (per patch) and `forcedSourceTable_tract` (per tract).

File: data-products-dp0-2/index.rst [hunk 2]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
+     - Forced photometry on the individual PVIs at the locations of all detected objects and sources (38 columns).
+   * - DiaObject
+     - diaObjectTable_tract
+     - Derived summary parameters for DiaSources associated by sky location, including lighcurve statistics (137 columns).
+   * - DiaSource
+     - diaSourceTable_tract
```
Comment (jeffcarlin) [inline data-products-dp0-2/index.rst:134]: This one has `diaSourceTable` (per visit) and `diaSourceTable_tract`. However, it doesn't really make sense to have DIASources at a tract level -- is this one a mistake?

File: data-products-dp0-2/index.rst [hunk 3]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
+     - Astrometric and photometric measurements for sources detected in the difference images (66 columns).
+   * - ForcedSourceOnDiaObject
+     - forcedSourceOnDiaObjectTable_tract
```
Comment (jeffcarlin) [inline data-products-dp0-2/index.rst:137]: There is also `forcedSourceOnDiaObjectTable`, which is per-patch.

File: data-products-dp0-2/index.rst [hunk 4]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
 
 **Principal Columns:**
 For convenience, Rubin Observatory staff have identified the principal columns which are most likely to be useful.
 These principal columns will be pre-selected in the Table View of the RSP's Portal Aspect.
-The table schema below also identify the principal columns.
 
 **Recommended Search Parameter "detect_isPrimary = True":**
-A good default search query parameter for catalogs of objects in the coadded images is to set **detect_isPrimary** = **True**, i.e., for the dp01_dc2_catalogs.position (TAP) and deepCoadd_ref (Butler) catalogs.
+A good default search query parameter for the Object, Source, and ForcedSource catalogs is to set **detect_isPrimary** = **True**.
 The ``detect_isPrimary`` parameter is ``True`` if a source has no children, is in the inner region of a coadd patch, is in the inner region of a coadd tract, and is not “detected” in a pseudo-filter.
```
Comment (jeffcarlin) [inline data-products-dp0-2/index.rst:154]: See [this link](https://pipelines.lsst.io/getting-started/multiband-analysis.html#filtering-for-unique-deblended-sources-with-the-detect-isprimary-flag) for more info?

**Review by jeffcarlin — APPROVED** [review]
Some minor changes to consider, but otherwise good to go!

File: data-products-dp0-2/index.rst [hunk 5]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
+
+.. list-table:: Catalog data available for DP0.2.
+   :widths: 100 100 390
+   :header-rows: 1
+
+   * - TAP Name
+     - Butler Name
+     - Description
+   * - Object
+     - objectTable
```
[the remark this answers is not in the document]
Comment (ktlim) [reply data-products-dp0-2/index.rst:122]: For real DRPs, we won't be keeping both of these (or the other pairs mentioned).  I would expect that only the more-aggregated versions would be part of the DR.

File: data-products-dp0-2/index.rst [hunk 6]
```diff
@@ -52,44 +52,41 @@ For DP0.2, the Rubin Data Production team will reprocess the same images with th
...
 
 Images
 ------
 
-For DP0.2 the DC2 data set has only two types of images: coadds and processed visit images.
-For DP0.2, images are only accessible with the Butler via the Notebook Aspect.
+The three main types of images available for DP0.2 are processed visit images, coadded images, and difference images.
 
-**Processed Visit Images** (PVIs):
+**Processed Visit Images** (PVIs; **calexps**):
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:67]: done

File: data-products-dp0-2/index.rst [hunk 7]
```diff
@@ -52,44 +52,41 @@ For DP0.2, the Rubin Data Production team will reprocess the same images with th
...
+A single CCD of a PVI is called a "calexp".
+
+There are many associated data products that are accessible alongside PVIs.
+These include the background ("calexpBackground") that was subtracted from the "calexp", which can be retrieved separately.
+Each PVI also has an associated mask plane that encodes quality and other information about each pixel,
+a WCS solution to be used in converting between pixel and sky coordinates,
+a photometric calibration object to be used in converting between fluxes and magnitudes for astronomical sources,
+and a model of the point-spread function (PSF) at each position on the image.
+
+**Coadded Images** (**deepCoadd**):
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:80]: also done

File: data-products-dp0-2/index.rst [hunk 8]
```diff
@@ -52,44 +52,41 @@ For DP0.2, the Rubin Data Production team will reprocess the same images with th
...
+An image that is the combination of multiple input images, often referred to as a "coadd" or a "deepCoadd".
+The input images have been aligned to a common projection and pixel grid;
+corrected to the same photometric scale, zero-point, and point-spread function (PSF);
 and had bad pixels, artifacts, and transient and variable object flux removed prior to combination.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:84]: yup

File: data-products-dp0-2/index.rst [hunk 9]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
     The center image is one tract quadrant, and the right image one hundredth the area of the tract quadrant. Patches are larger than the right image, as described in the DESC's paper:
     *"each tract is composed of 7 × 7 patches, and each patch is 4,100 × 4,100 pixels with a pixel scale of 0.2 arcsec"*.
 
-The first of the :ref:`DP0-2-Tutorials-Notebooks` demonstrates how to identify the tract and patch for a given coordinate, and retrieve and plot a coadd image.
-
-The `image display and manipulation tutorial <https://github.com/rubin-dp0/tutorial-notebooks/blob/main/03_Image_Display_and_Manipulation.ipynb>`_ demonstrates how to retrieve and display a coadd image,
-and to use its WCS and methods associated with the image to extract a cutout image zoomed in on a region of interest.
-
+**Difference Images**:
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:98]: Added (but not as part of the title b/c their names are long so its in the paragraph.

File: data-products-dp0-2/index.rst [hunk 10]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
+Any source detected in a difference image represents the *time-variable* flux component of the astrophysical object.
 
 .. _DP0-2-Data-Products-DPDD-Catalogs:
 
 Catalogs
 --------
 
-Source detection, measurement, and characterization have been run on both the PVIs and coadds to generate catalog data for DP0.2 (see also :ref:`Data-Processing-Overview-Coadded-Catalogs`).
+Source detection, measurement, and characterization have been run on the PVIs, coadds, and difference images to generate catalog data.
 Catalog data are accessible with the :ref:`Data-Access-Analysis-Tools-TAP` via the Portal or Notebook Aspect, and with the Butler via the Notebook Aspect.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:109]: removed

File: data-products-dp0-2/index.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:122]: It was mentioned underneath the table. Moved it to above. Added the note about real DRPs.

File: data-products-dp0-2/index.rst [hunk 11]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
+   :widths: 100 100 390
+   :header-rows: 1
+
+   * - TAP Name
+     - Butler Name
+     - Description
+   * - Object
+     - objectTable
+     - Astrometric and photometric measurements for objects detected in coadded images (990 columns).
+   * - Source
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:124]: Mentioned above the table now.

File: data-products-dp0-2/index.rst (hunk 1 above)
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:128]: ditto

File: data-products-dp0-2/index.rst (hunk 2 above)
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:134]: Not a mistake, this really exists, but, yes its the diaSourceTable that would be more useful to search so changed to reporting that in table.

File: data-products-dp0-2/index.rst (hunk 3 above)
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:137]: yes there is. changed to reporting that in table as main table

File: data-products-dp0-2/index.rst (hunk 4 above)
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:154]: added

File: data-products-dp0-2/index.rst [hunk 12]
```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
...
-However, because the catalog of sources detected in individual processed visit images (PVIs) is only available through the Butler, the Butler schema for several of the most useful DP0.2 catalogs are provided below.
-The Butler catalogs are listed in the approximate order that a processing workflow with the LSST Science Pipelines would generate them.
-For examples of how to access these Butler catalogs, see the :ref:`DP0-2-Tutorials-Notebooks` about the Butler.
-
-.. list-table:: Butler-accessible tables available for DP0.2.
-   :widths: 120 120 350
-   :header-rows: 1
+**Additional butler catalogs** can also be found by querying the collections in the butler registry.
+Some tables require different types of inputs: for example, "diaSourceTable" can be queried with a dataId that includes the visit,
+whereas "diaSourceTable_tract" can be queried with a dataId that includes the tract number.
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply data-products-dp0-2/index.rst:159]: yes real, also yes not emphasized anymore

**Pull request merged**